stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/QueryChannelsControllerImpl.kt

```diff
@@ -202,7 +202,6 @@ internal class QueryChannelsControllerImpl(
             output = result
             if (result.isSuccess) {
```

Comment:
Shouldn't we move the whole if into `runQueryOnline`? I think we won't call `updateChannelsAndQueryResults` after recovery which updates channels LiveData

Reply:
Yes, you're right. I did it for the online/offline execution of a query. But there is some duplication in the recovery flow. It's not so easy to decouple right now. I think we can make some refactoring there, but not right now.